srcpkgs/upwork/template

```diff
@@ -1,33 +1,41 @@
 # Template file for 'upwork'
 pkgname=upwork
-version=5.1.0.562
+version=5.3.3.862
 revision=1
 _rawver=${version//./_}
-_hashver="f3wgs5ljinabm69t"
+_hashver="0gofuuatmjqd0n86"
 archs="x86_64"
 depends="alsa-lib gtk+ GConf gdk-pixbuf nss libXtst libXScrnSaver"
 short_desc="Desktop Upwork app"
 maintainer="Arthur Evstifeev <mail@ap4y.me>"
-license="Proprietary"
-repository="nonfree"
-restricted=yes
-nopie=yes
+license="custom:Upwork"
 homepage="https://upwork.com/"
 distfiles="https://updates-desktopapp.upwork.com/binaries/v${_rawver}_${_hashver}/upwork_${version}_amd64.deb
  https://upwork.pactsafe.io/versions/5ab3efef63d65b6a18aab106.pdf>LICENSE.pdf"
-checksum="af70dae3d1cd5910d3ab591201caf75b13758d12e487b44cd32452c305df68c0
+checksum="249c3e168db5cf8bb46e2282357c4cdd8a8b80b9c0d599b8b787863b03f9f45e
  0949da5999c3b948bcc4165e6c55522915d9bab1790d27120eda0915aae143bb"
+repository="nonfree"
+restricted=yes
+nopie=yes
+
+# Skip runtime dependenies check, otherwise NEEDED libGLESv2.so (which is bundled with the package)
+# will trigger dependency on rpi-userland packages (makes no sense on x86_64).
+skiprdeps="/opt/${pkgname}/libEGL.so"
 
 do_extract() {
 	ar x ${XBPS_SRCDISTDIR}/${pkgname}-${version}/upwork_${version}_amd64.deb
-	tar xf data.tar.xz
+	bsdtar xf data.tar.xz
 }
 
 do_install() {
+	# Install to /opt instead of /usr/share since it contains shared libs
```

Comment:
no, not `/opt`. shove it into something like `/usr/lib/upwork` instead

Reply:
@Piraty just updated the package to 5.3.3.871, which has everything in `usr/share` moved to `opt` by upstream. Unlike having shared libs in `usr/share` (like previous version did) it doesn't seem to violate any standards or policies. Does it make sense to keep it as provided by upstream from now?